.gitignore

```diff
@@ -63,4 +63,5 @@ dist
 !.elasticbeanstalk/*.global.yml
 
 # AWS DOcker build
-ecr.repo
\ No newline at end of file
+ecr.repo
+/dps-gradle-spring-boot-suppressions.xml
```

Comment:
did you mean to add this here?  Shouldn't we be deleting `dependency-check-suppress-spring.xml` and using this instead?

Reply:
I think you still need to add that to gitignore though right? Else the build will keep adding it

Reply:
yes, but we don't want it in `.gitignore`, we want to use the one from the plugin instead

Reply:
That is the one from the plugin (in gitignore)